R/MetaAnalysis.R

```diff
@@ -0,0 +1,424 @@
+# Copyright 2021 Observational Health Data Sciences and Informatics
+#
+# This file is part of SkeletonComparativeEffectStudy
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#' Title
+#'
+#' @param allDbsFolder    Folder on the local file system containing the individual zip files across databases (i.e., sites)
+#' @param maExportFolder  A local folder where the meta-analysis results will be written. If not specified, results will be
+#'                        written to same directory with all other results.
+#' @param maxCores        Maximum number of CPU cores to be used when computing the meta-analyses.
+#' @param method          The meta-analysis method to use.  Options are "bayesian" or "DL" (DerSimonian-Laird).
```

Comment:
'bayesian' might be a bit misleading, since it is both Bayesian *and* using non-normal approximation. We may want to add Bayesian with normal approximations later. So maybe 'BayesianNonNormal'? (Also Bayes is a name, just like DerSimonian, so I tend to capitalize that.

Reply:
Of course, reverend Bayes himself :P
But updated to the suggested, more descriptive term